drivers/network/ndis/ndis/hardware.c

```diff
@@ -185,7 +251,22 @@ NdisReadPciSlotInformation(
     IN  ULONG       Length)
 {
   PLOGICAL_ADAPTER Adapter = NdisAdapterHandle;
+  NTSTATUS Status;
+
   /* Slot number is ignored since W2K for all NDIS drivers. */
+  Status = NdisQueryPciBusInterface(Adapter);
```

Comment:
Please consider moving this call to `NdisIAddDevice()` or `NdisIPnPStartDevice()`.